projboard/tests/test_like.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from projboard.models import Like, User, Article, Subject
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+NAME = "Shalev"
+
+
+@pytest.mark.django_db
+class TestLikeModel:
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_subjects(self):
+        sup = Subject(name='N')
+        sup1 = Subject(name='R')
+        sup.save()
+        sup1.save()
+        return [sup, sup1]
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_user(self):
+        user = User(name=NAME, nickname="S99", email="Shalev@gmail.com", password="Shalev123")
+        user1 = User(name="John", nickname="J99", email="John@gmail.com", password="John123")
+        user.save()
+        user1.save()
+        return [user, user1]
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_articles(self, create_user, create_subjects):
+        articles_list = []
+        for i in range(len(create_user)):
```

Comment:
Please refer the above comment use parameters

Reply:
Acutely I don't need many articles, I did not use them.
I will fix that